### DISCUSSION TITLE
q-date mask problem or bug

### DISCUSSION THREAD

**Discussion by gwinnem** [quasarframework/quasar#13167] [General - Components / Directives / etc] [answered]
**Background**
Im trying to change the mask in a q-date component as pr the [documentation](https://quasar.dev/vue-components/date#model-mask)
the mask im trying to use is: DD.MM.YYYY
the date im selecting is: 03 04 2022

**Problem**
The output then is: 
![image](https://user-images.githubusercontent.com/6427120/163172027-403e881c-76d7-4119-b5e7-22bd5ad93189.png)

**Expected output**
03.04.2020

**Comment by metalsadman** [comment]
Should be ##.##.####

**Comment by pratik227** [comment, marked as answer]
https://codepen.io/Pratik__007/pen/MWrqVzO

You need to remove the mask="date" from QInput or change it to ##.##.####

https://github.com/quasarframework/quasar/issues/4270

**Reply by gwinnem** [reply]
Hi pratik. Tnxs for the quick reply and for the codepen.
I unmarked your answer as a answer. The reason for it is that the date commponent shows the red icon when you set a date.
Do you also know how to avoid that ?

Geirr

**Reply by pratik227** [reply]
Bcz It's applying 'date' rules on the Input that's why :rules="['date']" and for this default it compares with "####/##/##"

**Reply by gwinnem** [reply]
Tnxs pratik
